src/legacy/core_plugins/kibana/public/discover/controllers/discover.js

```diff
@@ -220,7 +220,6 @@ function discoverController(
   $scope.minimumVisibleRows = 50;
   $scope.fetchStatus = fetchStatuses.UNINITIALIZED;
   $scope.refreshInterval = timefilter.getRefreshInterval();
-  $scope.savedQuery = $route.current.locals.savedQuery;
```

Comment:
You're correct that this is a leftover line mistakenly left in. We originally resolved the savedQuery like index patterns but decided against it.